openapi.yaml

```diff
@@ -10615,6 +10618,300 @@ components:
         - errorText
         - errorDetail
       title: API Error
+    WhatsApp:
```

Comment:
Please tag this schema with `Messages` to have it displayed in the Message API group:
```
x-tags:
  - Messages
```

Reply:
Added x-tags to WhatsApp